### PR TITLE
closes #2821 Text area and Text input docs

### LINKED ISSUE #2821: Theme docs - 'border' structure for TextArea and TextInput
The format for documenting theme properties would best be the last detail, sometimes the documentation is lumping together properties

Current situation:
TextArea and TextInput docs are:

```
'global.control.border': {
    description: 'The border of the textarea.',
    type: 'object',
    defaultValue: {
      width: '1px',
      radius: '4px',
      color: 'border',
    },
  },
```

desired behavior like Button

```
 'button.border.color': {
    description: `The color of the border.`,
    type: 'string | { dark: string, light: string }',
  },
  'button.border.radius': {
    description: 'The corner radius.',
    type: 'string',
    defaultValue: '18px',
  },
  'button.border.width': {
    description: 'The border width.',
    type: 'string',
    defaultValue: '2px',
  },
```

### PR DESCRIPTION
**Pull request by IanKBovard**
What does this PR do?
Adds to textArea and textInput docs.

Where should the reviewer start?
textArea/docs.js and textInput/docs.js

What testing has been done on this PR?
None

What are the relevant issues?
#2821

Do the grommet docs need to be updated?
auto

Should this PR be mentioned in the release notes?
no

Is this change backwards compatible or is it a breaking change?
backwards compatible

### CODE REVIEW

**Comment by IanKBovard** [comment]
use this branch instead of https://github.com/grommet/grommet/pull/2899

File: src/js/components/TextArea/doc.js [hunk 1]
```diff
@@ -44,6 +44,29 @@ Only use this when the containing context provides sufficient affordance.`,
 };
 
 export const themeDoc = {
+  'global.colors.border': {
+    description: 'The color of the border.',
+    type: 'object',
```
Comment (ShimiSun) [inline src/js/components/TextArea/doc.js]:
```suggestion
    type: 'string | { dark: string, light: string }',
```

Comment (IanKBovard) [reply src/js/components/TextArea/doc.js]: changes made

**Comment by ShimiSun** [comment]
Thank you for your contribution!

**Pull request merged**